cg/models/validation_cases/validation_case_data.py

```diff
@@ -0,0 +1,118 @@
+        LOG.info(f"Validation Data checks completed for {self.case_id}")
+
+    @staticmethod
+    def _validation_sample_name(sample: Sample) -> str:
+        """Return a new validation sample identifier. The identifier removes "ACC" and prepends "VAL"."""
+        return sample.internal_id.replace("ACC", "VAL")
+
+    @property
+    def validation_case_name(
+        self,
+    ) -> str:
+        """Return a case name with _validation appended."""
+        return f"{self.case_name}_validation"
+
+    def get_samples_to_copy(self) -> list[Sample]:
+        """
+        Return samples associated to a case.
+        """
+        case: Case = self.status_db.get_case_by_internal_id(self.case_id)
+        return [sample.internal_id for sample in case._get_samples]
+
+    def get_case_to_copy(self) -> Case:
+        """
+        Check if a case exists in StatusDB.
+            Raises: ValueError
+        """
+        case: Case = self.status_db.get_case_by_internal_id(self.case_id)
+        if not case:
+            raise ValueError(f"Case {self.case_id} not found in StatusDB.")
+        return case
+
+    def _generate_validation_samples(
+        self,
+        original_sample: Sample,
+    ) -> Sample:
+        """
+        Generate a validation sample record for StatusDB.
+        The new sample contains the original sample internal id and meta data
+        """
+        application_version: ApplicationVersion = self._get_application_version(original_sample)
+        sample_name: str = self._validation_sample_name(original_sample)
+        validation_sample: Sample = self.status_db.add_sample(
+            name=sample_name,
+            internal_id=sample_name,
+            sex=original_sample.sex,
+            order=original_sample.order,
+            from_sample=original_sample.internal_id,
+            tumour=original_sample.is_tumour,
+            priority=Priority.standard,
+            customer=original_sample.customer,
+            application_version=application_version,
+            received=original_sample.received_at,
+            prepared_at=original_sample.prepared_at,
+            last_sequenced_at=original_sample.last_sequenced_at,
+        )
+        return validation_sample
+
+    def get_validation_samples(self):
+        return [self._generate_validation_samples(sample) for sample in self.original_samples]
+
+    def _generate_validation_case(
+        self,
+    ) -> Case:
+        """
+        Generate a case for the validation samples.
+        The new case uses existing case data.
+        Customer will be set to cust000.
+        """
+        if self.status_db.case_with_name_exists(case_name=self.validation_case_name):
+            raise ValueError(f"Case with name {self.case_name} already exists.")
+        validation_case: Case = self.status_db.add_case(
+            data_analysis=self.original_case.data_analysis,
+            data_delivery=DataDelivery.NO_DELIVERY,
```

Comment:
These two fields should be part of the CLI options when generating the new validation cases. If not provided, they could default to using the original cases data analysis and `NO_DELIVERY`

Reply:
Any other options we should be able to select when creating new validation cases that you think we would need? @fevac @mathiasbio

Reply:
Not that I can think of! There is more information that could be added for future automatic validations, such as what to validate for the specific case we're adding but I don't think we're ready for that yet!

Reply:
I guess the only thing I can think of, but for which I cannot remember a use-case for at the moment, is the option to add the level of downsampling for the new samples created for the validation-case. 

If I understand it correctly. In case someone needed a downsampled sample specifically for a validation, they would with the current solution need to downsample it first using the downsample feature, which would create the downsampled case, and then specify that downsampled case here, which if I understood it correctly will create 2 additional new samples. If we could do the downsampling here already it could save some time and space 🤷

Reply:
As cli options I think it's enough with data analysis and data delivery. If any, adding/appending a comment but I don't use it and I don't know if someone does.

Then other options that should be not copied from the original case are: customer (cust000), action (hold), priority (standard?)